services/config_service.rb

```diff
@@ -54,18 +54,9 @@ def octokit_projects(provider_credential: nil)
       logger.debug("Getting code host for #{provider_credential.ci_user.email}, #{provider_credential.type}")
       current_code_hosting_service = self.code_hosting_service(provider_credential: provider_credential)
 
-      # current set of `GitRepoConfig.name`s that `provider_credential` has access to
-      current_repo_git_url_set = current_code_hosting_service.repos.map(&:html_url).to_set
-      # TODO: we have to improve repo handling, as it seems like we either have to implement
-      # proper paging, or we ask for specific repos instead
-      # Either way, my account has access to too many repos, so for now, let's just workaround using this
-      current_repo_git_url_set << "https://github.com/nakhbari/Hello"
-      current_repo_git_url_set << "https://github.com/taquitos/ci-sample-repo"
-      current_repo_git_url_set << "https://github.com/fastlane/ci"
-
       logger.debug("Finding projects we have access to with #{provider_credential.ci_user.email}, #{provider_credential.type}")
       projects = self.project_service.projects.select do |project|
-        current_repo_git_url_set.include?(project.repo_config.git_url)
+        current_code_hosting_service.access_to_repo?({repo: project.repo_config.git_url})
```

Comment:
`access_to_repo?({repo: project.repo_config.git_url})` can be:
`access_to_repo?(repo: project.repo_config.git_url)`

Reply:
done